src/mapping_navigation/scripts/navigation.py

```diff
@@ -12,17 +13,19 @@ class Navigation:
     """
 
     def __init__(self):
-        self.steering_pub = rospy.Publisher('lka/steering', Float64, queue_size=10)
-        self.curve_pub = rospy.Publisher('curveData', String, queue_size=10)
+        self.steering_pub = rospy.Publisher('steering', Float64, queue_size=10)
```

Comment:
Is removing "lka" from the topic name changing any behaviour? Wondering since other locations still use "lka/steering", for example lane_keep_assist.py (there are other files that also use "lka/steering" but they are from the previous year's project)

Reply:
yeah I moved to using our defined steering topic, I highly doubt we will user any of their steering data, and we defined it on the wiki

Reply:
Ill change the one in our lka too